R/cp04ossm_metbk.Rmd

```diff
@@ -0,0 +1,103 @@
+---
+title: "Pioneer METBK Example"
+output: html_notebook
+---
+
+This R Notebook provides an example on how to access telemetered
+meteorological data from the Coastal Pioneer Offshore Surface Mooring 
+from the beginning of August 2021 to the current day.

```

Comment:
I personally think you should add your name as author of this document.  Up to you though.
At the very least it should have a date in case it every becomes out dated information.